scripts/get-orig-tarball

```diff
@@ -298,14 +295,44 @@ dl_to_dir() {
         return 1
     }
 
-    orig=$(find_orig "$src" "$uver" "$TEMP_D") || {
+    upstream_orig=$(find_orig "$src" "$uver" "$TEMP_D") || {
         error "dget succeeded, but no orig tarball found."
         cat "$TEMP_D/${src}_${ver}.dsc" 1>&2
         ls $TEMP_D 1>&2;
         cd "$sdir"
         return 1
     }
 
+    if orig=$(find_orig "$src" "$uver" "$odir"); then
+        ORIG_SUM=$(sha256sum "${orig}" | awk '{print $1}' )
+        UPSTREAM_SUM=$(sha256sum "${upstream_orig}" | awk '{print $1}' )
+        if [ "${ORIG_SUM}" == "${UPSTREAM_SUM}" ]; then
+            error ${MSG_BORDER_LINE}
+            error "No difference between upstream orig.tar.gz and local."
+            error "No changes made to ${orig}"
+            error ${MSG_BORDER_LINE}
+            _RET="${odir}/${orig##*/}"
+            return 0
+        fi
+        error ${MSG_BORDER_LINE}
+        error " orig tarball existed at $orig and differs from upstream"
```

Comment:
Should these lines have leading spaces? (following the opening quotes)

Reply:
I was originally thinking of indenting scope on the msgs between MSG_BORDER_LINE. But it looks fine without leading padding. Best to keep it the same either way, and resolve inconsistencies.